collector/hadoop/README.md

```diff
@@ -0,0 +1,124 @@
+
+---
+# Ingest metrics using the Hadoop integration
+
+The OTEL Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
+{: .callout}
+
+## Requirements
+
+* OpenTelemetry Collector Contrib v0.53.0+
+
+## Prerequisites
+
+You must have a Lightstep Observability [access token](/docs/create-and-manage-access-tokens) for the project to report metrics to.
+
+## To enable JMX in Hadoop
+
+* [hadoop-env.sh](/collector/hadoop/conf/hadoop-env.sh)
+```sh
+export HDFS_NAMENODE_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=8004 $HDFS_NAMENODE_OPTS"
+export HDFS_DATANODE_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=8006 $HDFS_DATANODE_OPTS"
+```
+
+* [yarn-env-sh](/collector/hadoop/conf/yarn-env.sh)
+```sh
+export YARN_RESOURCEMANAGER_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=8002 $YARN_RESOURCEMANAGER_OPTS"
+export YARN_NODEMANAGER_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=8002 $YARN_NODEMANAGER_OPTS"
+```
+
+
+## Running the Example
+
+You can run this example with `docker-compose up` in this directory. You'll want to view this in Lightstep with a dashboard. 
+
+#### Hadoop Docker
+
+### Quick Start
+
+To deploy an example HDFS cluster, run:
+``` sh
+  docker compose up
+```
+
+Run example wordcount job:
+``` sh
+  make wordcount
+``
+
+`docker-compose` creates a docker network that can be found by running `docker network list`, e.g. `dockerhadoop_default`.
+
+Run `docker network inspect` on the network (e.g. `hadoop_integrations`) to find the IP the hadoop interfaces are published on. Access these interfaces with the following URLs:
+
+* Namenode: http://<dockerhadoop_IP_address>:9870/dfshealth.html#tab-overview
+* History server: http://<dockerhadoop_IP_address>:8188/applicationhistory
+* Datanode: http://<dockerhadoop_IP_address>:9864/
+* Nodemanager: http://<dockerhadoop_IP_address>:8042/node
+* Resource manager: http://<dockerhadoop_IP_address>:8088/
+
```

Comment:
I presume this is intended to mean that it's accessible like this in Docker DNS, so when coming from a container in the network. But I think there's benefit to being explicit as the operator requests would be resolved on the host.

Also I recommend checking that the example actually makes these ports accessible.